crates/rattler/src/install/entry_point.rs

```diff
@@ -12,8 +12,8 @@ use std::{fs::File, io, io::Write, path::Path};
 /// Get the bytes of the windows launcher executable.
 pub fn get_windows_launcher(platform: &Platform) -> &'static [u8] {
     match platform {
-        Platform::Win32 => unimplemented!("32 bit windows is not supported for entry points"),
-        Platform::Win64 => include_bytes!("../../resources/launcher64.exe"),
+        Platform::Win32 => include_bytes!("../../resources/cli-32.exe"),
```

Comment:
Shouldn't these only be included in their respective targets?

Reply:
No, because you might want to build a win-32 env on win-64, or a win-64 env on linux (and send it to a cloud machine, use pixi-pack, ...)

Reply:
Ah oke